docs/developer-guide/pprofiling.md

```diff
@@ -0,0 +1,148 @@
+# Performance Profiling in zot
+
+> :point_right: Use zot's built-in profiling tools to collect and analyze runtime performance.
+
+The profiling capabilities within zot allow a zot administrator to collect and export a range of diagnostic performance data such as CPU intensive function calls, memory allocations, and execution traces. The collected data can then be analyzed using go tools and a variety of available visualization tools.
```

Comment:
would add a link from "administrator" to authN/authZ content

Reply:
Done: "...allow a zot [administrator](../articles/authn-authz.md) to collect..."